### PR TITLE
Database init

### LINKED ISSUE #1: Backend database setup
## Description
Set up the database connection for the API following [this tutorial](https://fastapi.tiangolo.com/tutorial/sql-databases/#create-model-attributescolumns).

## Tasks
- [x] install sqlalchemy, if not already installed
- [x] add database URL
- [x] add the engine (use a sqlite DB)
- [x] Create `SessionLocal` class
- [x] Create the base class

### PR DESCRIPTION
**Pull request by ATayls**
Create the initial db.py and models.py boiler plate.

Dependancy function to get db for use in fastapi endpoints. see fastapi docs.

- Closes #1

### CODE REVIEW

File: iog-api/iog_api/models.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+
+from db import Base
+
+
+class Schemas(Base):
+    __tablename__ = 'projects'
```
Comment (dcl10) [inline iog-api/iog_api/models.py]: not "schemas"?

**Review by dcl10 — CHANGES_REQUESTED** [review]

File: iog-api/iog_api/models.py (hunk 1 above)
Comment (kuraisle) [reply iog-api/iog_api/models.py]: not "schemata"?

Comment (ATayls) [reply iog-api/iog_api/models.py]: pushed a change to __tablename__

schemata or schema?

**Review by dcl10 — APPROVED** [review]

File: iog-api/iog_api/models.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+
+from db import Base
+
+
+class Schemas(Base):
+    __tablename__ = 'schemas'
+    id = Column(Integer, primary_key=True)
+    name = Column(String, unique=True)
+    schema = Column(String)
```
Comment (kuraisle) [inline iog-api/iog_api/models.py:10]: Is it worth making the Column the JSON type?
https://docs.sqlalchemy.org/en/20/dialects/sqlite.html

**Review by kuraisle — APPROVED** [review]

**Pull request merged**